prow/scripts/library.sh

```diff
@@ -202,6 +202,18 @@ EOF
 
 }
 
+function runTestLogCollector(){
+    if [[ -n "${ENABLE_TEST_LOG_COLLECTOR}" && "${ENABLE_TEST_LOG_COLLECTOR}" == true ]] ; then
+        if [[ "$BUILD_TYPE" == "master" ]]; then
+            log::info "Install test-log-collector"
+            export PROW_JOB_NAME="${TEST_LOG_COLLECTOR_PROW_JOB_NAME:-some_prow_job}"
+            (
+                "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/install-test-log-collector.sh" || true # we want it to work on "best effort" basis, which does not interfere with cluster
+            )
+        fi
+    fi
+}
+
```

Comment:
keep in mind that we want to remove `library.sh` in favour of the `lib` directory. If there's a possibility please refrain from editing this script in the future.

Reply:
I'm not super familiar with the library plans. May I leave it as I implemented it here? I think it's quite ok solution and it doesn't create much boilerplate code there. If not, I'm open for suggestions, @Ressetkk  :D

Reply:
No worries. I'm planning to work on removing the `library.sh` from now on. Deprecation of it is purely for the sake of maintaining the smaller scripts instead of a big one. You can leave it here